packages/data-point/lib/resolve/reducer-path.js

```diff
@@ -19,6 +19,15 @@ function resolveObjectPath (acc, jsonPath) {
     return _.get(acc, jsonPath.slice(2))
   }
 
+  if (jsonPath.slice(-2) === '[]') {
```

Comment:
this function originally only accepts the reducer.name, I would suggest you pass the full reducer value, so you have access to the other properties of the reducer path.

this way here you can check on `reducer.asCollection` instead of line 22

Reply:
Some of the reducers that are passed in are not objects though. I can add a test for the different scenarios (I'm seeing strings, undefined, and objects with the current tests), but it might be simpler to pass in the `asCollection` property from the reducer in the `resolve` method directly

```javascript
function resolve (store, resolveReducer, accumulator, reducerPath) {
  const value = resolveObjectPath(accumulator, reducerPath.name, reducerPath.asCollection)
  return Promise.resolve(utils.set(accumulator, 'value', value))
}
```

This will be falsy in all those scenarios without additional work

Reply:
In the ending I'm passing in the whole `reducerPath` and destructuring the `type` and the `asCollection` and checking that it's a `ReducerPath` and that `asCollection` is truthy before parsing out the array value.